### PR TITLE
POLIO-1504 POLIO-1505: renaming obr name and GPEI coordinator

### PR DESCRIPTION
**Pull request by beygorghor**
Campaign Modal view - Base info tab - rename "OBR name" into "Campaign identifier"

Campaign Modal view - Base info tab - rename "GPEI coordinator" into "Responsible officer"

Related JIRA tickets : POLIO-1504 POLIO-1505

## Self proofreading checklist

- [ ] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- added campaign type id and name to campaign detail call
- added one context to pass props to tabs component
- change GPEI coordinator and OBR name lable if campaign is not of type polio

## How to test

Open campaign modal on polio and non polio one's and check labels

## Print screen / video

Screenshot 2024-03-14 at 15 38 43
Screenshot 2024-03-14 at 15 38 36

### CODE REVIEW

File: plugins/polio/js/src/constants/translations/fr.json [hunk 1]
```diff
@@ -145,6 +146,7 @@
     "iaso.polio.form.label.re_submitted_to_orpg_wider": "Resoumis à ORPG wider group",
     "iaso.polio.form.label.re_submitted_to_rrt": "Resoumis à to RRT",
     "iaso.polio.form.label.repeatBudgetStep": "Renvoyer l'étape pour corrections ou comme rappel",
+    "iaso.polio.form.label.responsibleOfficer": "Officier responsable",
```
Comment (hakifran) [inline plugins/polio/js/src/constants/translations/fr.json:149]: @beygorghor is this translation match the context? what if we use 'Agent responsable'?

**Review by hakifran — APPROVED** [review]
Everything is good for me, apart one translation in french which I don't know if it matched the context **Officier responsable**

**Comment by hakifran** [comment]
@beygorghor there is conflict in plugins/polio/js/src/domains/Campaigns/MainDialog/CreateEditDialog.tsx

File: plugins/polio/js/src/constants/translations/fr.json (hunk 1 above)
Comment (beygorghor) [reply plugins/polio/js/src/constants/translations/fr.json:149]: Cool thanks, much better !!

**Pull request merged**